Sources/Markdown/Interpretive Nodes/Aside.swift

```diff
@@ -192,3 +192,27 @@ public struct Aside {
         self.content = Array(newBlockQuote.blockChildren)
     }
 }
+
+extension BlockQuote {
+    /// Conservatively checks the text of this block quote to see whether it can be parsed as an aside.
+    ///
+    /// Whereas ``Aside/init(_:)`` will use all the text before the first colon in the first line,
+    /// or else return an ``Aside`` with a ``Aside/Kind-swift.struct`` of ``Aside/Kind/note``,
```

Comment:
This documentation link should be unambiguous already 
```suggestion
    /// or else return an ``Aside`` with a ``Aside/Kind`` of ``Aside/Kind/note``,
```